builtin/plugins/coin/coin.go

```diff
@@ -42,16 +43,18 @@ type (
 	Account              = ctypes.Account
 	InitialAccount       = ctypes.InitialAccount
 	Economy              = ctypes.Economy
-
-	BurnRequest = ctypes.BurnRequest
+	Policy               = ctypes.Policy
+	BurnRequest          = ctypes.BurnRequest
 )
 
 var (
-	ErrSenderBalanceTooLow = errors.New("sender balance is too low")
+	ErrSenderBalanceTooLow   = errors.New("sender balance is too low")
+	InvalidBaseMintingAmount = errors.New("Base Minting Amount should be greater than zero")
```

Comment:
Go naming convention for errors is `ErrBlahBlah`

Reply:
Done.